src/main/java/org/fundacionjala/sfdc/core/driver/RemoteBrowserConnection.java

```diff
@@ -18,7 +19,10 @@
     protected static final String ACCESS_KEY = PROPERTIES_MANAGER.getRemoteAccessKey();
     protected static final String RESOLUTION = "resolution";
     private final String url;
-    private static final Logger LOGGER = Logger.getLogger(RemoteBrowserConnection.class.getName());
+
+    private static final Logger LOGGER = LogManager.getLogger(RemoteBrowserConnection.class.getName());
+
+    //private static final Logger LOGGER = Logger.getLogger(RemoteBrowserConnection.class.getName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)